src/components/dashboard/FeedList.js

```diff
@@ -54,6 +54,12 @@ const FeedList = ({
   const feeds = data && data instanceof Array && data.length ? data : [emptyFeed]
   const flRef = createRef()
 
+  /**
+   * Creating this constant with useMemo inside the function's body because of errors with
+   * Hot Module Replacement. 
+   * As AnimatedSwipeableFlatList doesn't support changes on the property viewabilityConfig between renders,
+   * if it's defined without being memoized, it gets recreated on every hot reload and fails.
+   */
   const VIEWABILITY_CONFIG = useMemo(
```

Comment:
we didnt have any issues with web hot reload before, why do we now have a problem with this swipeableflatlist?

Reply:
Just checked master and works fine. Not sure why it doesn't in react-native branch. Might be because of differences in webpack configurations.

Another weird thing I noticed is that https://github.com/gaearon/react-hot-loader says

> If you need hooks support, use @hot-loader/react-dom

but `@hot-loader/react-dom` doesn't seem to be used in master